sql/_34_fig/cbrd_24419/cases/cbrd_24419_1.sql

```diff
@@ -0,0 +1,121 @@
+/*
+ * This is a test case to verify the CBRD-24419 issue.
+ *
+ * The result should be the same as before.
+ *   - db_class
+ *   - db_direct_super_class
+ *   - db_vclass
+ *   - db_attribute
+ *   - db_attr_setdomain_elm
+ *   - db_method
+ *   - db_meth_arg
+ *   - db_meth_arg_setdomain_elm
+ *   - db_meth_file
+ *   - db_index
+ *   - db_index_key
+ *   - db_auth
+ *   - db_trig
+ *   - db_partition
+ *   - db_stored_procedure
+ *   - db_stored_procedure_args
+ *   - db_server
+ *   - db_synonym
+ *   - skip checking db_charset, db_collation doesnt have owner_name attribute.
```

Comment:
db_charset and db_collation in the system views also changed the query specification.
So we have to check whether the result is the same by testing.

It has nothing to do with the owner_name.